src/main/java/mate/academy/spring/model/CinemaHall.java

```diff
@@ -4,12 +4,18 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
 
 @Entity
 public class CinemaHall {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @NotNull
```

Comment:
Why do we use constraints for models? Maybe better to use it in Dtos? And for String variables better to use @NotBlank instead  @NotNull

Reply:
@NotBlank